CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+## How to contribute to Accentor
+
+#### **Did you find a bug?**
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/accentor/web/issues).
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/accentor/web/issues/new). We have provided a template to help you report bugs.
+
+#### **Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+* Be sure to check that all github actions complete succesfully.
+
+#### **Do you intend to add a new feature or change an existing one?**
+
+* Be sure to make an [issue with a feature request first](https://github.com/accentor/web/issues/new). If the feature requires changes on the API, be sure to submit [an issue there first](https://github.com/accentor/api/issues/new).
+
+* You can always discuss a certain solution in the comments under an issue or in our [matrix channel](https://matrix.to/#/!PCYHOaWItkVRNacTSv:vanpetegem.me?via=vanpetegem.me&via=matrix.org).
+
+#### **Do you have questions about Accentor?**
+
+* Open an issue or ask any question about how to use Accentor in the [matrix channel](https://matrix.to/#/!PCYHOaWItkVRNacTSv:vanpetegem.me?via=vanpetegem.me&via=matrix.org).
```

Comment:
I'd prefer not to allow opening an issue to ask questions. Once GitHub discussions is released we can allow questions there.